src/test/java/com/google/cloud/spanner/jdbc/ExecuteMockServerTest.java

```diff
@@ -320,7 +320,7 @@ public void testPreparedStatementExecuteUpdate() throws SQLException {
     try (Connection connection = createConnection()) {
       assertEquals(1, connection.prepareStatement(DML).executeUpdate());
       // TODO: Enable the next statement once PreparedStatement supports executing DDL through the
-      // executeUpdate() method.
+      //       executeUpdate() method.
```

Comment:
Please check if these blank spaces were added unintentionally..

Reply:
Yes, those are intentional. They make sure that the second line of the comment is part of the TODO, and is highlighted as such in the IDE.